[Bulk] Broken Link fixes (#878)
MicrosoftDocs/biztalk-docs: biztalk/technical-guides/checklist-optimizing-performance-on-hyper-v.md

> |                                                                                      Step                                                                                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    Reference                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |

Reviewer: ```suggestion
| Step | Reference |```

> |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

Reviewer: Delete unecessary whitespace:

```suggestion
|---|---|
```

> |                      Scope the hardware requirements for the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] solution.                       | -   Follow the guidance in the “Planning the Environment for BizTalk Server” section of the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] Operations Guide [here](https://go.microsoft.com/fwlink/?LinkId=122399) to scope the hardware requirements for the solution.<br />-   To scope the edition and number of BizTalk Servers that will be required by the solution review BizTalk Server planning considerations documented in “Planning the BizTalk Server Tier” [here](https://go.microsoft.com/fwlink/?LinkId=122401).<br />-   To scope the version and number of [!INCLUDE[btsSQLServerNoVersion](../includes/btssqlservernoversion-md.md)] computers that will be required by the solution, review database planning considerations documented in “Planning the Database Tier” [here](https://go.microsoft.com/fwlink/?LinkId=122402) and the “Performance Overhead of Running SQL Server in Hyper-V” section of the “Running SQL Server 2008 in a Hyper-V Environment Best Practices and Performance Considerations” whitepaper available [here](https://go.microsoft.com/fwlink/?LinkId=144622).<br />-   To complete planning for development, testing, staging, and production environments review “Planning the Development, Testing, Staging, and Production Environments” [here](https://go.microsoft.com/fwlink/?LinkId=122403). |

Reviewer: Fixed "[here]" link text, changed curly quotes to plain quotes, and fixed table formatting for easier readability:

```suggestion
| Scope the hardware requirements for the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] solution. | - To scope the hardware requirements for the solution, follow the guidance in the "Planning the Environment for BizTalk Server" section in the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] [Operations Guide](https://go.microsoft.com/fwlink/?LinkId=122399). <br><br>- To scope the edition and number of BizTalk Servers that will be required by the solution review BizTalk Server planning considerations documented in "Planning the BizTalk Server Tier" [here](https://go.microsoft.com/fwlink/?LinkId=122401). <br><br>- To scope the version and number of [!INCLUDE[btsSQLServerNoVersion](../includes/btssqlservernoversion-md.md)] computers that the solution requires, review the database planning considerations in the following documentation: <br>--- ["Planning the Database Tier"](https://go.microsoft.com/fwlink/?LinkId=122402) <br>--- The "Performance Overhead of Running SQL Server in Hyper-V" section in the ["Running SQL Server 2008 in a Hyper-V Environment Best Practices and Performance Considerations" whitepaper](https://go.microsoft.com/fwlink/?LinkId=144622). <br><br>- To complete planning for development, testing, staging, and production environments, review ["Planning the Development, Testing, Staging, and Production Environments"](https://go.microsoft.com/fwlink/?LinkId=122403). |
```

> | After scoping the hardware requirements of your BizTalk Server solution, plan to configure the Hyper-V machines with **110% – 125%** of the CPU and disk resources if possible. |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            For example, if the hardware requirements for a physical BizTalk Server computer used by the solution is determined to be 2GB RAM, a dual core CPU running at 2GHZ, and 2x 500 GB physical disks, then ideally, the Hyper-V virtual machine used by the solution would be configured with 2 or more virtual processors running >= 2.2 GHZ, and faster physical disks (typically by adding spindles or using faster disks).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |

Reviewer: ```suggestion
| After you scope the hardware requirements of your BizTalk Server solution, plan to configure the Hyper-V machines with 110% - 125% of the CPU and disk resources if possible. |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            For example, if the hardware requirements for a physical BizTalk Server computer used by the solution is determined to be 2GB RAM, a dual core CPU running at 2GHZ, and 2x 500 GB physical disks, then ideally, the Hyper-V virtual machine used by the solution would be configured with 2 or more virtual processors running >= 2.2 GHZ, and faster physical disks (typically by adding spindles or using faster disks).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

## Optimize Hyper-V Performance
 Use the following general guidelines to configure Hyper-V for optimal performance.

> |Step|Reference|

Reviewer: ```suggestion
| Step | Reference |
```

> |----------|---------------|

Reviewer: ```suggestion
|---|---|
```

> |Apply recommended guidance for performance tuning virtualization servers. **Note:**  For the test scenarios described in [Testing BizTalk Server Virtualization Performance](~/technical-guides/testing-biztalk-server-virtualization-performance.md), the configuration options that were applied are described in the “Physical Infrastructure Specifics” and the “Virtualization Specifics” sections of the [Test Scenario Overview](~/technical-guides/test-scenario-overview.md) topic.|“Performance Tuning for Virtualization Servers” section of the “Performance Tuning Guidelines for Windows Server 2008 R2” document available [here](/windows-server/administration/performance-tuning/).|

Reviewer: ```suggestion
| Apply recommended guidance for performance tuning virtualization servers. <br><br>**Note**:  For the test scenarios described in [Testing BizTalk Server Virtualization Performance](~/technical-guides/testing-biztalk-server-virtualization-performance.md), the configuration options that were applied are described in the "Physical Infrastructure Specifics" and the "Virtualization Specifics" sections in the article, [Test Scenario Overview](~/technical-guides/test-scenario-overview.md). | "Performance Tuning for Virtualization Servers" section in the article, ["Performance Tuning Guidelines for Windows Server 2008 R2"](/windows-server/administration/performance-tuning/). |
```

> |Close any Virtual Machine Connection windows that aren’t being used.|The Virtual Machine Connection window(s) displayed when double-clicking a virtual machine name in the Hyper-V manager consume resources that could be otherwise utilized.|

Reviewer: ```suggestion
| Close any Virtual Machine Connection windows that aren't in use. | In the Hyper-V manager, when you double-select a virtual machine name, Virtual Machine Connection windows appear. These windows consume resources that could otherwise be free for use. |
```

|                                                                    Windows Vista  32-bit                                                                    |            2            |

> [!NOTE]

> >  For more information about the guest operating systems that are supported on Hyper-V, see [here](https://go.microsoft.com/fwlink/?LinkID=118347).

Reviewer: Existing link goes to no info:

```suggestion
> For more information about the guest operating systems that are supported on Hyper-V, see [Supported Windows guests](/virtualization/hyper-v-on-windows/about/supported-guest-os) and [Supported Windows guest operating systems for Hyper-V on Windows Server](/windows-server/virtualization/hyper-v/supported-windows-guest-operating-systems-for-hyper-v-on-windows).
```

### Optimize Disk Performance

>  Follow these guidelines to optimize disk performance of guest operating systems running in a Hyper-V virtual environment:

Reviewer: ```suggestion
Follow these guidelines to optimize disk performance of guest operating systems running in a Hyper-V virtual environment:
```
